growth.cpp

```diff
@@ -172,7 +172,7 @@ void plotVTU(int time_step, Node* head)
 	outFile << "<?xml version=\"1.0\"?>\n";
 	outFile << "<VTKFile type=\"UnstructuredGrid\" version=\"0.1\" byte_order=\"LittleEndian\">\n";
 	outFile << "\t<UnstructuredGrid>\n";
-	outFile << "\t\t<Piece NumberOfPoints=\"" << nodes << "\" NumberOfCells=\"" << nodes - 1 << "\">\n";
+	outFile << "\t\t<Piece NumberOfPoints=\"" << nodes << "\" NumberOfCells=\"" << "1" << "\">\n";

```

Comment:
should be a variable you pass in. Fine for now but will be interesting to see how this goes when we go to 2 cells.